release/release_tests.yaml

```diff
@@ -3759,7 +3759,7 @@
     timeout: 7200
     script: python many_nodes_tests/actor_test.py
     wait_for_nodes:
-      num_nodes: 251
+      num_nodes: 241
```

Comment:
Can we revert this change for now? We will do some discussion this Friday, and decide how to proceed this